<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_Setup</class>
 <widget class="QDialog" name="Dialog_Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>408</width>
    <height>262</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../rrcc.qrc">
    <normaloff>:/png/png/setup.png</normaloff>:/png/png/setup.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <widget class="QWidget" name="tab_communication">
      <attribute name="icon">
       <iconset resource="../rrcc.qrc">
        <normaloff>:/png/png/communication.png</normaloff>:/png/png/communication.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Communication</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>IP-Address</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string/>
                </property>
                <property name="pixmap">
                 <pixmap resource="../rrcc.qrc">:/png/png/network.png</pixmap>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_ip">
                <property name="toolTip">
                 <string>IP-Address of Robot</string>
                </property>
                <property name="text">
                 <string notr="true">255.255.255.255</string>
                </property>
                <property name="maxLength">
                 <number>15</number>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_4">
           <property name="title">
            <string>Message-ID</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/msgid.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="spinBox_id">
              <property name="toolTip">
               <string>Message-ID Counter</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
              <property name="showGroupSeparator" stdset="0">
               <bool>true</bool>
              </property>
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="maximum">
               <number>1000000</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>AES-Token</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../rrcc.qrc">:/png/png/padlock.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_token">
            <property name="minimumSize">
             <size>
              <width>230</width>
              <height>0</height>
             </size>
            </property>
            <property name="toolTip">
             <string>AES-Token to encrypt Messages</string>
            </property>
            <property name="text">
             <string notr="true"/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="cursorPosition">
             <number>0</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_ssh">
            <property name="toolTip">
             <string>Extract Token via SSH (Root required)</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rrcc.qrc">
              <normaloff>:/png/png/gift.png</normaloff>:/png/png/gift.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_convert">
            <property name="toolTip">
             <string>Convert 96 / 16 Byte to 32 Byte Token</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rrcc.qrc">
              <normaloff>:/png/png/converter.png</normaloff>:/png/png/converter.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_sound">
      <attribute name="icon">
       <iconset resource="../rrcc.qrc">
        <normaloff>:/png/png/sound.png</normaloff>:/png/png/sound.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Sound</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Volume</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../rrcc.qrc">:/png/png/volume.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="horizontalSlider_volume">
            <property name="toolTip">
             <string>Voice Volume</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="tickPosition">
             <enum>QSlider::TicksBothSides</enum>
            </property>
            <property name="tickInterval">
             <number>5</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_volume">
            <property name="text">
             <string notr="true">0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_9">
         <item>
          <widget class="QGroupBox" name="groupBox_dnd">
           <property name="toolTip">
            <string>Enable/Disable Do Not Disturb Mode</string>
           </property>
           <property name="title">
            <string>Do Not Disturb</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_10">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/mute.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dnd1">
              <property name="toolTip">
               <string>Starting Time</string>
              </property>
              <property name="displayFormat">
               <string notr="true">HH:mm</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="timeEdit_dnd0">
              <property name="toolTip">
               <string>Ending Time</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_6">
           <property name="title">
            <string>Voice</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/voice.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_sound_install">
              <property name="toolTip">
               <string>Install Voice Package</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="icon">
               <iconset resource="../rrcc.qrc">
                <normaloff>:/png/png/open.png</normaloff>:/png/png/open.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_sound_pack">
              <property name="toolTip">
               <string>Create Voice Package from wav Files</string>
              </property>
              <property name="text">
               <string notr="true"/>
              </property>
              <property name="icon">
               <iconset resource="../rrcc.qrc">
                <normaloff>:/png/png/compress.png</normaloff>:/png/png/compress.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="toolButton_sound_unpack">
              <property name="toolTip">
               <string>Extract wav Files from Voice Package</string>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="icon">
               <iconset resource="../rrcc.qrc">
                <normaloff>:/png/png/decompress.png</normaloff>:/png/png/decompress.png</iconset>
              </property>
              <property name="iconSize">
               <size>
                <width>24</width>
                <height>24</height>
               </size>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_misc">
      <attribute name="icon">
       <iconset resource="../rrcc.qrc">
        <normaloff>:/png/png/misc.png</normaloff>:/png/png/misc.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Misc</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QGroupBox" name="groupBox_carpet">
         <property name="toolTip">
          <string>Enable/Disable Carpet Mode</string>
         </property>
         <property name="title">
          <string>Carpet Mode</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QSpinBox" name="spinBox_carpet_integral">
            <property name="toolTip">
             <string>Parameter Integral</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_carpet_high">
            <property name="toolTip">
             <string>Parameter High</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_carpet_low">
            <property name="toolTip">
             <string>Parameter Low</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_carpet_stalltime">
            <property name="toolTip">
             <string>Parameter Stalltime</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_carpet_reset">
            <property name="toolTip">
             <string>Reset all Parameter to Defaults</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rrcc.qrc">
              <normaloff>:/png/png/reset.png</normaloff>:/png/png/reset.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QGroupBox" name="groupBox_9">
           <property name="title">
            <string>Online Update</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <widget class="QCheckBox" name="checkBox_update">
              <property name="text">
               <string>search Update on Start</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="title">
            <string>Reserved</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_ssh">
      <attribute name="icon">
       <iconset resource="../rrcc.qrc">
        <normaloff>:/png/png/ssh.png</normaloff>:/png/png/ssh.png</iconset>
      </attribute>
      <attribute name="title">
       <string notr="true">SSH</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_13">
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>Username</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/user.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ssh_username">
              <property name="toolTip">
               <string>SSH Username</string>
              </property>
              <property name="text">
               <string notr="true">root</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_ssh_password">
           <property name="toolTip">
            <string>Enable/Disable Login via Password</string>
           </property>
           <property name="title">
            <string>Password</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/pin.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ssh_password">
              <property name="toolTip">
               <string>SSH Password</string>
              </property>
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_ssh_keyfile">
         <property name="toolTip">
          <string>Enable/Disable Login via Key</string>
         </property>
         <property name="title">
          <string>Keyfile</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string/>
            </property>
            <property name="pixmap">
             <pixmap resource="../rrcc.qrc">:/png/png/safe.png</pixmap>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ssh_keyfile">
            <property name="toolTip">
             <string>SSH private Key</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_ssh_keyfile_passphrase">
            <property name="toolTip">
             <string>SSH private Key Passphrase</string>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::PasswordEchoOnEdit</enum>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="toolButton_ssh_keyfile">
            <property name="toolTip">
             <string>Select SSH Keyfile</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../rrcc.qrc">
              <normaloff>:/png/png/open.png</normaloff>:/png/png/open.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_wifi">
      <attribute name="icon">
       <iconset resource="../rrcc.qrc">
        <normaloff>:/png/png/wifi.png</normaloff>:/png/png/wifi.png</iconset>
      </attribute>
      <attribute name="title">
       <string>WiFi</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Credentials</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6" stretch="0,0,0,0">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/ssid.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_ssid">
              <property name="toolTip">
               <string>SSID for WiFi Connection</string>
              </property>
              <property name="text">
               <string>&lt;SSID&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="bottomMargin">
             <number>5</number>
            </property>
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap resource="../rrcc.qrc">:/png/png/key.png</pixmap>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineEdit_key">
              <property name="toolTip">
               <string>Key for WiFi Connection</string>
              </property>
              <property name="text">
               <string>&lt;KEY&gt;</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 0, 0);</string>
            </property>
            <property name="text">
             <string>Set new credentials will change your device token!</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rrcc.qrc"/>
 </resources>
 <connections/>
</ui>
